scripts/bootstrap.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
-
+mv /tmp/start-sql-config.txt /etc/yum.repos.d/mysql-community.repo
 dnf -y update
+dnf -y install mysql-community-server

```

Comment:
You can "merge" this line with line 7 and move the systemctl commands after line 7.